Build.mak

```diff
@@ -19,10 +19,14 @@ $O/libdyaml.a: $(shell find $C/submodules/d-yaml/source -type f -name '*.d') \
 
 ################################################################################
 
+$O/allunittests $O/fastunittest: $O/libvibed.a $O/libdyaml.a
+
 $O/%unittests: override LDFLAGS += -L$O -lvibed -ldyaml -levent -lssl -lcrypto
 $O/pkg-neptune.stamp: release overview
 
-%unittests: $O/libvibed.a $O/libdyaml.a
+ITFLAGS += --tmp=$O --bin=$B
+integrationtest: $B/neptune-release $O/libvibed.a
```

Comment:
To be strictly correct, this should also be set per `$O/test-*`, which is quite annoying. We could leave it like this as in practice it will only fail if someone tries to build one individual test, but if you want to do it right, you could do something like:

```make
$(patsubst $T/$(INTEGRATIONTEST)/%/main.d,$O/test-%.stamp,\
		$(filter-out $(TEST_FILTER_OUT),\
		$(wildcard $T/$(INTEGRATIONTEST)/*/main.d))): $B/neptune-release $O/libvibed.a
```

This might be something to add to MakD. Feel free to create an issue there.

For now you could also just list the tests manually, it might be simpler and more clear:
```make
$O/test-initial_release: $B/neptune-release $O/libvibed.a
```

Reply:
Update: 
* made sure test turns of gpg
* updated Build.mak